clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -18,8 +18,14 @@
         "url": "https://github.com/eclipse/che-che4z-lsp-for-cobol/issues"
     },
     "keywords": [
-		"mainframe", "lsp", "cobol", "z/os", "zos", "language-server", "jcl"
-	],
+        "mainframe",
+        "lsp",
+        "cobol",
+        "z/os",
+        "zos",
+        "language-server",
+        "jcl"
+    ],
```

Comment:
These changes are not part of this activity

Reply:
changes reverted